### ISSUE TITLE
Changes needed to update

### ISSUE THREAD

**Issue by SteveALee** [w3c/wai-resource-template#2]
Updates required given changes made recently to get all resources up-to date

Blocked by #4

[notes](https://wai-website-theme.netlify.app/technical/new-resource/)
[example](https://github.com/w3c/wai-about-wai)

- [x] netlify.toml - remove old section, update others
- [x] transfer Netlify site to W3C team and update settings
- [x] README.md
- [x] Gemfile + gemfile.lock - note: check .lock is OK!
- [x] .git/workflows/init - looks like broken WIP helping automation of setting up new repo, doesn't handle navigation
- [x] _data/navigation.yml this is a file not link into _external - bug!? See below
- [x] _external - submodule to old commit  - update
- [x] [Shawn's content update](https://github.com/w3c/wai-resource-template/pull/1) NB above action is invoked on push and breaks
- [x] w3c.json
- [x] use wai-gems

Questions for @slhenry and perhaps @nitedog 

- [w3c.json](https://github.com/w3c/wai-resource-template/blob/master/w3c.json) - remove yatil and replace with shawn?
- navigation is a file not a link and yatils setup instructions say copy the latest. Why would this be? I see problems. - new issue #4

**Comment by slhenry** [comment]
> w3c.json - remove yatil and replace with shawn?

yes. thanks.
